tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_hashes/fresh_system.pass.sh

```diff
@@ -2,6 +2,11 @@
 #
 # profiles = xccdf_org.ssgproject.content_profile_pci-dss
 
-# we expect the system is in good shape
-# TODO: add check here, so this script fails when not true
-true
+# verify all packages from rpm 
+result=$(rpm -Va | grep '^..5')
+
+if [ -z "$result" ]; then
```

Comment:
this is a bash anti-pattern. you are running `[` which returns 0 exit code or 1, 0 for true. then you check that it's 0 (true) and then you run true wihch again exists with 0. you can simplify this to just running `[` and that's all.